.github/workflows/harvest-all.yml

```diff
@@ -0,0 +1,6421 @@
+  #     name: Extract vuls-data-extracted-oss-fuzz-osv@main
+  #     needs: [fetch-oss-fuzz-osv]
+  #     if: ${{ ( success() || failure() ) }}
+  #     uses: ./.github/workflows/extract-vanilla.yml
+  #     with:
+  #       target: oss-fuzz-osv
+  #       branch: main
+
+  #   extract-nightly-oss-fuzz-osv:
+  #     name: Extract vuls-data-extracted-oss-fuzz-osv@nightly
+  #     needs: [extract-oss-fuzz-osv]
+  #     if: ${{ ( success() || failure() ) }}
+  #     uses: ./.github/workflows/extract-vanilla.yml
+  #     with:
+  #       target: oss-fuzz-osv
+  #       branch: nightly
+
+  harvest-post-raw-oss-fuzz-osv:
+    name: Harvest vuls-data-raw-oss-fuzz-osv
+    needs: [check, fetch-oss-fuzz-osv]
+    if: ${{ ( success() || failure() ) && needs.check.outputs.remainder == 0 }}
+    uses: ./.github/workflows/harvest-post-raw-main.yml
+    with:
+      tag: vuls-data-raw-oss-fuzz-osv
+      pack-threads: 2
+      pack-windowMemory: 4g
+
+  #   harvest-post-extracted-oss-fuzz-osv:
+  #     name: Harvest vuls-data-extracted-oss-fuzz-osv
+  #     needs: [check, extract-nightly-oss-fuzz-osv]
+  #     if: ${{ ( success() || failure() ) && needs.check.outputs.remainder != 0 }}
+  #     uses: ./.github/workflows/harvest-post-extracted.yml
+  #     with:
+  #       tag: vuls-data-extracted-oss-fuzz-osv
+  #       pack-threads: 2
+  #       pack-windowMemory: 4g
+
+  fetch-paloalto-list:
+    name: Fetch vuls-data-raw-paloalto-list
+    uses: ./.github/workflows/fetch-vanilla.yml
+    with:
+      target: paloalto-list
+
+  #   extract-paloalto-list:
+  #     name: Extract vuls-data-extracted-paloalto-list@main
+  #     needs: [fetch-paloalto-list]
+  #     if: ${{ ( success() || failure() ) }}
+  #     uses: ./.github/workflows/extract-vanilla.yml
+  #     with:
+  #       target: paloalto-list
+  #       branch: main
+
+  #   extract-nightly-paloalto-list:
+  #     name: Extract vuls-data-extracted-paloalto-list@nightly
+  #     needs: [extract-paloalto-list]
+  #     if: ${{ ( success() || failure() ) }}
+  #     uses: ./.github/workflows/extract-vanilla.yml
+  #     with:
+  #       target: paloalto-list
+  #       branch: nightly
+
+  harvest-post-raw-paloalto-list:
+    name: Harvest vuls-data-raw-paloalto-list
+    needs: [check, fetch-paloalto-list]
+    if: ${{ ( success() || failure() ) && needs.check.outputs.remainder == 0 }}
+    uses: ./.github/workflows/harvest-post-raw-main.yml
+    with:
+      tag: vuls-data-raw-paloalto-list
+      pack-threads: 2
+      pack-windowMemory: 4g
+
+  #   harvest-post-extracted-paloalto-list:
+  #     name: Harvest vuls-data-extracted-paloalto-list
+  #     needs: [check, extract-nightly-paloalto-list]
+  #     if: ${{ ( success() || failure() ) && needs.check.outputs.remainder != 0 }}
+  #     uses: ./.github/workflows/harvest-post-extracted.yml
+  #     with:
+  #       tag: vuls-data-extracted-paloalto-list
+  #       pack-threads: 2
+  #       pack-windowMemory: 4g
+
+  fetch-paloalto-json:
```

Comment:
Does this need vuls-data-raw-paloalto-list?

Reply:
fixed: https://github.com/vulsio/vuls-data-db/compare/13f900f7c04514c90d72788e929a9a69df4dc780..9934b8a5441db7ff9d53ac3f53db0ffbc4e32b5e